### PR TITLE
ci: add github.com ecdsa fingerprint to know_host

### PR DESCRIPTION
**Pull request by 5d**
*Issue #, if available:*

*Description of changes:*

- add `github.com` ECDSA finger print to know_host

*Check points:*

- [ ] ~Added new tests to cover change, if needed~
- [X] All unit tests pass
- [X] All integration tests pass
- [ ] ~Updated CHANGELOG.md~
- [ ] ~Documentation update for the change if required~
- [X] PR title conforms to conventional commit style

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by atierian** [comment]
I'm sure that I'm missing something obvious, but why does [switching to shallow clones](https://github.com/aws-amplify/aws-sdk-ios/pull/4623) make this change necessary?

**Comment by 5d** [comment]
@atierian , there is an SSH fingerprint error in step [prepare_release_sdk](https://app.circleci.com/pipelines/github/aws-amplify/aws-sdk-ios/7053/workflows/c3e0f9e6-c31e-4ac0-8775-5f9009d3508b/jobs/158705) for `main` branch. This step is using `circleci/android` docker image and it hangs while doing git clone. We add the fingerprint also make it more robust at security point of view.

**Review by royjit — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -688,6 +695,7 @@ jobs:
   create_pullrequest_for_modelupdate:
     docker:
       - image: circleci/android:api-27-alpha
+    working_directory: /tmp/project
```
Comment (royjit) [inline .circleci/config.yml:698]: this required?

Comment (5d) [reply .circleci/config.yml:698]: it is required for `git clone` to work properly.

**Pull request merged**